assignment/index.html

```diff
@@ -58,25 +58,49 @@
 
       /* =====================
 
-      Start code
+      Start c
 
       ===================== */
 
       var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
-
-      /* =====================
-
-      End code
-
-      ===================== */
-
-
-      jsonToCsv(healthCenters);
-      addMarkers(map);
-    </script>
-    <!--Your code ends here-->
-  </body>
-</html>
+        let HospitalLocation = [['LNG','LAT','OBJECTID','NAME','ORGANIZATION','ZIP','PHONE','WEBSITE','DENTAL_PHONE','FULL_ADDRESS']]
+        for (var i=1; i<healthCenters.length; i++) {
+            var GetArray = [parseFloat(`${healthCenters[i].LNG}`),
+                     parseFloat(`${healthCenters[i].LAT}`),
```

Comment:
What does `parseFloat()` do here? It could convert strings to numbers, but aren't these already numbers?